### PR TITLE
Add performance.now polyfill.

### PR DESCRIPTION
**Pull request by adrigzr**
[fix] add performance.now polyfill.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,5 +2,24 @@
 'use strict';
 
 module.exports = {
-	name: 'ember-task-scheduler'
+	name: 'ember-task-scheduler',
+
+	importPolyfill: function(app) {
+		var polyfill = 'vendor/performance-polyfill.js';
+
+		if (this.import) {  // support for ember-cli >= 2.7
```
Comment (josex2r) [inline index.js]:
```js
const import = this.import || app.import;

if (import) {
 import(polyfill, { prepend: true });
} else {
  console.warn('Please run: ember install ember-cli-import-polyfill');
}
```

File: index.js [hunk 2]
```diff
@@ -2,5 +2,24 @@
...
+			this.import(polyfill, { prepend: true });
+		} else if (app.import) { // support ember-cli < 2.7
+			app.import(polyfill, { prepend: true });
+		} else {
+			console.warn('Please run: ember install ember-cli-import-polyfill')
```
Comment (josex2r) [inline index.js]: semicolon

File: index.js (hunk 1 above)
Comment (adrigzr) [reply index.js]: You break function context that way.

File: index.js (hunk 2 above)
Comment (adrigzr) [reply index.js]: Done.

**Pull request merged**